assets/scss/helpers/_classes.scss

```diff
@@ -0,0 +1,130 @@
+.micro {
```

Comment:
Great to have this file to place classes, even here we could put the atomic classes.
These classes I think were copied from another side, anyway, none of these classes are currently being used, we could erase them.
Great job!

Reply:
if this is a atomic class, it should do only 1 thing